src/flex-table.cpp

```diff
@@ -347,11 +351,12 @@ void table_connection_t::stop(bool updateable, bool append)
             log_debug("Using GeoHash for clustering table '{}'",
                       table().name());
             if (table().geom_column().srid() == 4326) {
-                sql += "ST_GeoHash({},10)"_format(geom_column_name);
+                sql += fmt::format("ST_GeoHash({},10)",
+                                   table().geom_column().name());
             } else {
-                sql +=
-                    "ST_GeoHash(ST_Transform(ST_Envelope({}),4326),10)"_format(
-                        geom_column_name);
+                sql += fmt::format(
+                    "ST_GeoHash(ST_Transform(ST_Envelope({}),4326),10)",
+                    table().geom_column().name());
```

Comment:
Why remove the quotes?

Reply:
Looks like I botched the rebase. Fixed commit on the way...